com.github.tchx84.Flatseal.json

```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "com.github.tchx84.Flatseal",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "41",
+    "runtime-version": "master",
```

Comment:
For this, we will need to wait for GNOME 42...

Reply:
Ill update it either when libhandy 1.6 is released or when GNOME 42 is released.

Reply:
I added libhandy 1.5.90 to the manifest rather than setting it to master.